spec/calendar.html

```diff
@@ -834,22 +834,22 @@ <h1>
       <h1>
         ISOResolveMonth (
           _fields_: an Object,
-        ): either a normal completion containing an integer or a throw completion
+        ): either a normal completion containing ~unused~ or a throw completion
       </h1>
       <dl class="header">
         <dt>description</dt>
         <dd>
-          It validates that the *"month"* and *"monthCode"* properties of the given _fields_ are not inconsistent and merges them into an integer month which is returned, or throws an exception if both are *undefined*.
+          It validates that the *"month"* and *"monthCode"* properties of the given _fields_ are not inconsistent and merges them into a *"month"* property with a positive integral Number value in the inclusive interval from 1 to 12, or throws an exception if both are *undefined*.
```

Comment:
To me, "merges" implies that `monthCode` will be removed from the object, or at least is somewhat ambiguous about what will happen with `monthCode` after the AO returns. Maybe something like the following would be clearer?

```suggestion
          It ensures that the *"month"* property of _fields_, if *undefined*, is set to the integral Number value in the inclusive interval from 1 to 12 that corresponds to the *"monthCode"* property of  _fields_.
          If both properties are *undefined* or if the properties are inconsistent, or if either property is invalid, then a *RangeError* will be thrown.
```